code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -26,6 +26,33 @@
 	I.req_gun_tags = list(GUN_SILENCABLE)
 	I.prefix = "silenced"
 
+/obj/item/gun_upgrade/muzzle/pain_maker
+	name = "SA \"PainMaker\" muzzle"
+	desc = "A threaded barrel that can be attached to the muzzle of most projectile guns. \
+	Threaded barrel device made of a coil sensor and heater. As bullets pass the device they are slowed down and heated up by the coil, causing them to deform when hitting a target and imparting all their painful energy. \
+	Typically used when taking a hostages or kidnapping."
+	matter = list(MATERIAL_PLASTEEL = 3, MATERIAL_PLASTIC = 1)
+	icon_state = "silencer"
+	price_tag = 100
+
+/obj/item/gun_upgrade/muzzle/pain_maker/New()
+	..()
+	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
+	I.weapon_upgrades = list(
+		UPGRADE_BULK = 2,
+		GUN_UPGRADE_STEPDELAY_MULT = 1.5,
+		GUN_UPGRADE_DAMAGE_MULT = 0.8,
```

Comment:
```suggestion
		GUN_UPGRADE_DAMAGE_MULT = 0.5,
```
This should be a lot lower, as shown, to relaly put into effect that it should be used more with rubbers or sopo bullets and less with actual bullets. You can offset a 0.8 pretty easily by putting other mods on after it in different places.